test/cppsuite/src/main/database_operation.cpp

```diff
@@ -332,7 +332,7 @@ database_operation::remove_operation(thread_worker *tc)
              * one.
              */
             if (ret == WT_NOTFOUND) {
-                WT_IGNORE_RET_BOOL(tc->txn.commit());
+                (void)(tc->txn.commit());
```

Comment:
I'd recommend a new `testutil_ignore_ret` macro instead of void casts to align with our existing uses of `testutil_check` and `testutil_check_error_ok`